server/client.go

```diff
@@ -206,6 +206,7 @@ const (
 	DuplicateClientID
 	DuplicateServerName
 	MinimumVersionRequired
+	ClusterNamesIdentical
```

Comment:
Add this in monitor.go (ClosedState String()):
```
	case ClusterNamesIdentical:
		return "Cluster Names Identical"
```

Reply:
ok done, thanks.